### DISCUSSION TITLE
[#991930/983795] Chat and Coordinate UI on top of Editor UI

### DISCUSSION THREAD

**Discussion by fraysdev** [Mojang/minecraft-editor#65] [Bugs]
It kinda annoying that sometimes, when you ran a command, chat UI obstruct the Editor UI. Especially when you enabled "Show Coordinate".

**Comment by SmokeyStack** [comment]
Related to https://github.com/Mojang/minecraft-editor/discussions/43 ?

**Reply by minecraftchloe** [reply]
Totally related!

**Reply by fraysdev** [reply]
Oops, I didn't see it

**Comment by minecraftchloe** [comment]
Updated title to reflect internal tracking ID

**Comment by minecraftchloe** [comment]
Should be fixed in current build or our next release this week -- lmk if this pops up again as an issue!!
